### PR TITLE
Debriefing tool: TER catch / sample weight

### PR DESCRIPTION
**Pull request by Mak-NOAA**
Related to https://github.com/nwfsc-fram/boatnet/issues/2151
Added Catch weight / sample weight TER

### CODE REVIEW

File: util/tripChecks.ts [hunk 1]
```diff
@@ -267,7 +276,7 @@ function runBlankFitValueCheck(tripErrorDoc: WcgopTripError, trip: any, operatio
 
 
 //trip check code 110014 
-function runInactiveVesselCheck(tripErrorDoc: WcgopTripError, trip: any) {
+function runInactiveVesselCheck(tripErrorDoc: WcgopTripError, trip: WcgopTrip) {
     let error : WcgopError = {severity: Severity.error,
         description: 'Vessel inactive, please review selected vessel',
         dateCreated: moment().format(),
```
Comment (Mak-NOAA) [inline util/tripChecks.ts:282]: @sethgerou-noaa I added the object type in the parameters like WcgopTrip and on line 286 "errorValue: trip.vessel.vesselStatus.description," I got a typescript error of Property 'description' does not exist on type 'string' I'm guessing we need to make a change in vessel.d.ts right?

Comment (sethgerou-noaa) [reply util/tripChecks.ts:282]: @Mak-NOAA weird, I just came across this issue today as well.

The vessel model (https://github.com/nwfsc-fram/boatnet-module/blob/master/bn-models/src/models/_lookups/vessel.ts) has vesselStatus as a string, but vesselStatus has been ETLed as a partial object (not the whole vessel-status lookup doc).

To me the string makes more sense, as the object really doesn't add anything - it's just a description and a couch id.
We may need to discuss as a team - to decide which is correct.  Admittedly switching to a vesselStatus string would be more work...

If we go the string route:
We've made changes to some of the vessels, so re-ETLing would not be ideal.
I could write a script to replace vesselStatus.description with a vesselStatus string, but Nick would also need to re-ETL trips with updated vessels.

If we go the object route - we just need to update the model.

Thoughts?

**Review by sethgerou-noaa — APPROVED** [review]

File: util/tripChecks.ts (hunk 1 above)
Comment (Mak-NOAA) [reply util/tripChecks.ts:282]: @sethgerou-noaa I would prefer to just update the model so it matches the ETLing. That way the TypeScript won't be complaining. I'm just not sure how we go about updating the model. Can you commit that change?

Comment (sethgerou-noaa) [reply util/tripChecks.ts:282]: I've updated the vessel model and published to NPM.
You'll need to update "@boatnet/bn-models" in tripsApi's package.json to 0.5.9 and npm install.
Think you may also have to restart vs code for the  type error to update/go away.

Comment (Mak-NOAA) [reply util/tripChecks.ts:282]: Awesome, thanks @sethgerou-noaa that worked

**Pull request merged**